### PR TITLE
Optimize pair buffer in DefaultBroadPhaseBuffer

### PR DESCRIPTION
**Pull request by tommybuonomo**
Original [optimisation](https://github.com/jbox2d/jbox2d/pull/64) already done on jbox2d.

### CODE REVIEW

File: lib/src/collision/broadphase/default_broadphase_buffer.dart [hunk 1]
```diff
@@ -110,18 +111,18 @@ class DefaultBroadPhaseBuffer implements TreeCallback, BroadPhase {
     // Send the pairs back to the client.
     int i = 0;
     while (i < _pairCount) {
-      Pair primaryPair = _pairBuffer[i];
-      Object userDataA = _tree.getUserData(primaryPair.proxyIdA);
-      Object userDataB = _tree.getUserData(primaryPair.proxyIdB);
+      int primaryPair = _pairBuffer[i];
+      // print("Pairs => ${primaryPair >> 32} ${primaryPair & _mask}");
```
Comment (erickzanardo) [inline lib/src/collision/broadphase/default_broadphase_buffer.dart]: I think this was left here accidentally?

Comment (tommybuonomo) [reply lib/src/collision/broadphase/default_broadphase_buffer.dart]: Yes in fact, now fixed 👍

**Review by luanpotter — APPROVED** [review]
LGTM but I would wait for @spydon to make sure

**Review by spydon — APPROVED** [review]
Awesome, thanks for the contribution!

**Pull request merged**